callback_plugins/custom_junit.py

```diff
@@ -0,0 +1,128 @@
+from ansible.plugins.callback.junit import CallbackModule as JunitCallbackModule
+from ansible.plugins.callback.junit import HostData
+
+import os
+import time
+import re
+from ansible.utils._junit_xml import TestCase, TestError, TestFailure, TestSuite, TestSuites
+
+
+class CallbackModule(JunitCallbackModule):
+    """
+    Custom callback that overrides the default JUnit callback
+    """
+    CALLBACK_NAME = 'custom_junit'
+
+    def __init__(self):
+        super(CallbackModule, self).__init__()
+
+        # Custom environment variable handling
+        # self._output_dir = os.getcwd()
+        # Update this to parse these values from the config file, as well as the env.
+        self._output_dir = os.path.expanduser("~/.ansible.log")
+        # self._test_case_prefix = os.getenv('JUNIT_TEST_CASE_PREFIX', '[TEST]')
+        self._test_case_prefix = os.getenv('JUNIT_TEST_CASE_PREFIX', 'TEST')
+        #self._fail_on_ignore = os.getenv('JUNIT_FAIL_ON_IGNORE', 'False').lower()
```

Comment:
The commented lines of code above here seem like something maybe isn't finished. Are they needed? Is it some debugging leftover? I think we should delete them if there isn't a reason for them to stay.